includes/controls/settings-footer.php

```diff
@@ -0,0 +1,535 @@
+<?php
+namespace Elementor\Core\Kits\Documents\Tabs;
```

Comment:
???

Reply:
This namespace is an Elementor namespace. Hello Theme is not part of that namespace. Create a namespace for Hello Theme, like Hello_Elementor and use it everywhere that's necessary.

In line 16, where you extend `Tab_Base`, you should either
use the entire qualifier, like so:
`extends \Elementor\Core\Kits\Documents\Tabs\Tab_Base {`
or
add a `use` statement at the top after the namespace, like so:
`use \Elementor\Core\Kits\Documents\Tabs\Tab_Base`

Reply:
Same comments for settings-header.php